src/components/Card/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+export interface LinkProps {
+  label?: string;
+  url?: string;
+}
+
+const Link: React.FC<LinkProps> = ({ label, url }: LinkProps) => {
```

Comment:
I think that we don't need the `Link` component. Check my other review about the card changes. Also, one tip, if you need somehow to use the `<Link />` there is one that can be imported from `react-router-dom`. If you have any doubts about how to implement use it, check the sidebar component.

Reply:
ooh okay thank @rluders. I will check that.